Analysis/Tasks/PWGHF/taskLc.cxx

```diff
@@ -140,7 +145,13 @@ struct TaskLcMC {
         auto indexMother = RecoDecay::getMother(particlesMC, candidate.index0_as<aod::BigTracksMC>().mcParticle_as<soa::Join<aod::McParticles, aod::HfCandProng3MCGen>>(), pdg::Code::kLambdaCPlus, true);
         auto particleMother = particlesMC.iteratorAt(indexMother);
         registry.fill(HIST("hPtGenSig"), particleMother.pt()); // gen. level pT
-        registry.fill(HIST("hPtRecSig"), candidate.pt());      // rec. level pT
+        auto ptRec = candidate.pt();
+        registry.fill(HIST("hPtRecSig"), ptRec); // rec. level pT
+        if (candidate.originMCRec() == Prompt) {
```

Comment:
Please use explicit namespace qualifier: `OriginType::Prompt` instead of `Prompt`.